src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
```

Comment:
This is a code style issue. In your CSS, you've added both top and bottom margins to the `.search` class. It's recommended to be consistent with your margins and add only top or bottom, but not both, to avoid potential margin collapse.